olive/common/config_utils.py

```diff
@@ -129,7 +129,7 @@ def __len__(self):
 
 
 class ConfigDictBase(ConfigBase):
-    __root__: Dict[str, Any] = None
```

Comment:
should the ConfigListBase.__root__ be non-None?

Reply:
I think it is fine. It cannot be None but can be `{}` since the typing of `__root__` is Dict.
We should avoid to assign None to a Dict type, right?

Reply:
Sorry, updated the List to None as well.